README.md

```diff
@@ -1,2 +1,74 @@
-# tw-academy-s9
-Homework for the TW Academy
+# Sample Style Guide
+## Technical Writer Academy - May 2021
+---
+## Table of contents
+
+[Bold](#Bold)  
+[Italics](#Italics)  
+[Fixed Font](#Fixed-font)  
+[Abbreviations](#Abbreviations)  
+[Capitalization](#Capitalization)  
+[Acronyms](#Acronyms)  
+[Commas in Lists](#Commas-in-Lists)  
+[Numbers](#Numbers)  
+[Contractions](#Contractions)  
+[Apostrophes](#Apostrophes)   
+## Bold  
+Use bold to highlight button names, fields names, menus, radio buttons, or checkboxes. For example:  
+- Click the **Create** button to start the process.  
+- Enter your IP address in the **IP Address** field.  
+
+## Italics
+Use italics to emphasize a word or group of words. For example:
+- At least two reviews are required before you check in your code.
+- You must not enter without your ID card.  
+> **Important:** Use this style sparingly to avoid diluting it's strenght.  
+  
+  ## Fixed font  
+  Use a fixed font to highlight:
+  - **Commands or pieces of code.** For example:
+Run the ```npm install text-cli``` command to install the utility.  
+- **File or folder names.** For example:
+Read the `README.md` file before you proceed  
+## Abbreviations
+Never use the following abbreviations in a formal document:  
+|   |    |
```

Comment:
I think it was a good solution to leave the header row empty since the source document had no header. However, I would suggest as a best practice for tables always to have a header. You could propose something for this and the other table in the page.

Reply:
Thanks, the header was a headache, but I do think it's better with one.